extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads-tracking.js

```diff
@@ -0,0 +1,56 @@
+define('wikia.articleVideo.featuredVideo.adsTracking', [
+	'ext.wikia.adEngine.utils.eventDispatcher',
+	'ext.wikia.adEngine.video.player.jwplayer.jwplayerTracker'
+], function (eventDispatcher, tracker) {
+	function updateParams(params, currentAd) {
+		var wrapperCreativeId,
+			wrapperId;
+
+		if (currentAd) {
+			params.lineItemId = currentAd.getAdId();
+			params.creativeId = currentAd.getCreativeId();
+			params.contentType = currentAd.getContentType();
```

Comment:
What do you think about emptying those fileds on `adError`?

Reply:
Done